### DISCUSSION TITLE
How to make VS Code default editor for the changeset text?

### DISCUSSION THREAD

**Discussion by tordans** [changesets/changesets#1507] [Q&A]
I would like to use VS Code to write my changeset text. Like when I tell git to use VS Code to write my merge commit messages.

How can I do this?

---

Alternatively is there a way to retrieve the new changeset file name+path after running `npx changeset`? I could use this to write an empty changeset that I then re open in VSCode…

**Comment by Andarist** [comment]
When you invoke `changeset add` it prints:
> (submit empty line to open external editor)

So you can just press Enter to open an external editor. The editor to use is determined like this:
https://github.com/mrkmg/node-external-editor/blob/e1070a8295b8ebbafa7b29802ae0e2da0fad47c0/src/index.ts#L134-L139
